src/Conversion/ONNXToKrnl/RNN/GRU.cpp

```diff
@@ -171,6 +170,10 @@ getWeightPack<ONNXGRUOp, GruWeightPack>(
   Value R = op->R();
   // direction
   StringRef direction = op->direction();
+  // linear_before_reset.
+  bool linearBeforeReset = true;
```

Comment:
Isn't this just
```
linearBeforeReset = (op->linear_before_reset() != 0)

Reply:
Just the personal style to read the code smoothly. The short form somehow takes me a few seconds to understand.